electron/src/main/update.ts

```diff
@@ -0,0 +1,16 @@
+import log from 'electron-log/main';
+
+export function setupUpdates() {
+  setTimeout(() => {
+    const { updateElectronApp, UpdateSourceType } = require('update-electron-app');
```

Comment:
Interesting that this is a dynamic require. Is this package built into the distribution? I have no idea the packaging detects imports.

Reply:
Copy-paste from the `update-electron-app` example to test auto updater. Fixed to import.